core/src/main/java/io/lionweb/lioncore/java/serialization/data/SerializedChunk.java

```diff
@@ -13,7 +13,7 @@ public class SerializedChunk {
   private final Map<String, SerializedClassifierInstance> classifierInstancesByID = new HashMap<>();
 
   private String serializationFormatVersion;
-  private final List<UsedLanguage> languages = new ArrayList<>();
+  private final Set<UsedLanguage> languages = new LinkedHashSet<>();
```

Comment:
I would not use a set here for two reasons: 1) Preserving the order 2) Load faithfully files containing twice the same language

Reply:
`LinkedHashSet` #forTheWin !

Reply:
I would still think that in SerializedChunk it makes sense to allow for duplicates. If we don't we would not be able to deserialize and reserialize a file and get the same result

Reply:
I agree (even if `LinkedHashSet` preserves order modulo removals): we shouldn't have any "magic".